PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,119 @@
+# Quality Assurance Checklist
+To make reviews more efficient, please make sure the board meets the following standards and check everything off once the board meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._
+
+There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions.
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Minimum Prerequisites
+- [ ] The board passes ERC and DRC.
+    - Note some ERC warnings are acceptable (**_CAUTION_** please be careful)
+    - Note some DRC warnings are acceptable (less so than ERC: **_CAUTION_** please be careful).
+- [ ] All traces are routed.
+- [ ] Units are in metric.
+- [ ] Gerbers are in a zip file in the main directory (after physical design is done)
```

Comment:
The JLCPCB plugin for KiCAD now automates this very nicely; one click and it sends everything into a prod folder with the relevant positions and bom file needed for SMT assembly, if any.
I usually add a PDF of the layout and schematic and the ibom.html output to this folder too.

![image](https://user-images.githubusercontent.com/31521956/236358016-0c87a26e-6063-4549-ae36-68a22e2eac69.png)
![image](https://user-images.githubusercontent.com/31521956/236358028-6dcbe6c6-2b00-4808-b674-4fe2a018cb48.png)

Still figuring out how I like to version boards: my personal proposed workflow is to zip production into vX_X_X and then make a release on GH; the zip file gets updated every release instead of stashing a bunch of them (vX_X_0, vX_X_1, ...).

Reply:
Very easily could be turned into a gh workflow or gh action, but I'm not touching CICD pipelines again for few more years if i can help it

Reply:
For versioning, I just said that every master commit must be tagged as a version number. If they do want to check older versions, they just need to check out that commit. Ig that's the same as a release, but I've never saved the zip files of older boards.